includes/modules/pages/index/header_php.php

```diff
@@ -23,19 +23,21 @@
 $current_category_has_products = false;
 $current_category_has_subcats = false;
 if (isset($cPath) && zen_not_null($cPath)) {
-    $category_status_query = 
-        "SELECT categories_status
-           FROM " . TABLE_CATEGORIES . "
-          WHERE categories_id = :currentCategoryId
-          LIMIT 1";
-    $category_status_query = $db->bindVars($category_status_query, ':currentCategoryId', $current_category_id, 'integer');
-    $category_status = $db->Execute($category_status_query);
-    if ($category_status->EOF) {
-        $current_category_not_found = true;
-    } elseif ($category_status->fields['categories_status'] == '0') {
-        $current_category_is_disabled = true;
+    if ($cPath > 0) {

```

Comment:
Despite the changes in whitespace/indentation, the only change here is wrapping this section in the `if ($cPath > 0) {` wrapper.
For simpler review, click the "gear" icon in github's Diff screen and tick the checkbox to "ignore whitespace".